packages/xo-server/src/api/host.mjs

```diff
@@ -530,6 +531,25 @@ setControlDomainMemory.resolve = {
   host: ['id', 'host', 'administrate'],
 }
 
+// -------------------------------------------------------------------
+
+export async function isPubKeyTooShort({ host }) {
+  const certificate = await this.getXapi(host).call('host.get_server_certificate', host._xapiRef)
+
+  const cert = new X509Certificate(certificate)
+  return cert.publicKey.asymmetricKeyDetails.modulusLength < 2048
```

Comment:
Maybe you could use a constant that describes what 2048 is, and that can be reused if needed?
Ex: CERT_PUBKEY_MIN_SIZE

Reply:
Maybe you could use a constant that describes what 2048 is, and that can be reused if needed?
Ex: CERT_PUBKEY_MIN_SIZE